action.yaml

```
@@ -2,12 +2,22 @@ name: Install/Uninstall Keptn
 description: Install or uninstall Keptn inside a Kubernetes context
 inputs: 
   KEPTN_VERSION:   
-    description: The version of Keptn that should be installed (e.g. 0.13.1)
-    required: true
-  KEPTN_INSTALL_PARAMETERS:
-    description: Installation parameters for Keptn
+    description: The version of Keptn that should be installed (e.g. 0.16.0) if not specified latest available version will be used
     required: false
-    default: --endpoint-service-type=LoadBalancer
+    default: ""
+  HELM_VALUES:
+    description: Helm values to set during install passed as a YAML multiline string
+    required: false
+    default: |-
+      control-plane:
+        apiGatewayNginx:
+          type: LoadBalancer
+      continuous-delivery:
+        enabled: true
+  KEPTN_NAMESPACE:
+    description: Kubernetes namespace where Kptn will be installed
```

I think that could be a typo
```suggestion
    description: Kubernetes namespace where Keptn will be installed
```